addons/mail/static/src/models/dialog/dialog.js

```diff
@@ -8,6 +8,27 @@ registerModel({
     name: 'Dialog',
     identifyingFields: ['manager', ['attachmentViewer', 'followerSubtypeList']],
     recordMethods: {
+        /**
+         * @param {Element} element
+         * @returns {boolean}
+         */
+        isInContent(element) {
+            return Boolean(this.record && this.record.contains(element));
+        },
+        /**
+         * @private
+         * @returns {boolean|FieldCommand}
+         */
+        _computeIsCloseable() {
+            if (this.attachmentViewer) {
+                /**
+                 * Prevent closing the dialog when clicking on the mask when the user is
+                 * currently dragging the image.
+                 */
+                return !this.attachmentViewer.isDragging;
+            }
+            return clear();
```

Comment:
I still find this confusing to mix compute with default true and clear() to mean true.
I would prefer if there was no default and it was explicitly true here.
Default should ideally not be necessary on computed fields (the only that we have are when another compute uses the result of the first compute, and their order is wrong and having no default would crash the 2nd compute).